### PR TITLE
Adds interchangeable bar signs and a variety of choices.

### PR DESCRIPTION
**Pull request by Mandurrrh**
Re opening niknaks pr to port interchangeable bar signs. Player can click on the bar sign and choose from a  pop up menu one of many available ss13 themed bar signs. It randomly chooses one for round start but bar access will allow them to change it. Another change to make bartender job a bit more appealing. 

ss 2015-02-17 at 09 54 43

### CODE REVIEW

**Comment by Chocobro** [comment]
p cool! :+1:

**Comment by TheVekter** [comment]
On one hand, I like the idea of being able to choose what my bar is called.

On the other hand, it will -always- be the Maltese Falcon.

File: html/changelogs/mandurrh-barsign.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+################################
+# Mandurrh Changelog File
+#
+# 
+
+# Your name.  
+author: Mandurrh
+
+
```
Comment (optimumtact) [inline html/changelogs/mandurrh-barsign.yml]: missing the delete-after field

**Comment by xxalpha** [comment]
:+1:

File: code/game/objects/structures/barsigns.dm [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	icon = "thebark"
+	desc = "Ian's bar of choice"
+
+/datum/barsign/harmbaton
+	name = "The Harmbaton"
+	icon = "theharmbaton"
+	desc = "A great dining experience for both security members and assistants"
+
+/datum/barsign/thesingulo
+	name = "The Singulo"
```
Comment (Menshin) [inline code/game/objects/structures/barsigns.dm]: You forgot the .io

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 3]
```diff
@@ -2713,8 +2713,8 @@
 "bai" = (/obj/machinery/power/apc{cell_type = 2500; dir = 1; name = "Captain's Office APC"; pixel_y = 24},/obj/structure/cable{d2 = 8; icon_state = "0-8"},/turf/simulated/floor/wood,/area/crew_quarters/captain)
 "baj" = (/turf/simulated/floor/wood,/area/crew_quarters/captain)
 "bak" = (/obj/machinery/door/firedoor,/obj/machinery/door/airlock/glass{name = "Diner"},/turf/simulated/floor/plasteel,/area/crew_quarters/bar)
-"bal" = (/obj/structure/sign/maltesefalcon/left,/turf/simulated/wall,/area/crew_quarters/bar)
-"bam" = (/obj/structure/sign/maltesefalcon/right{pixel_y = 0},/turf/simulated/wall,/area/crew_quarters/bar)
+"bal" = (/obj/structure/sign/barsign,/turf/simulated/wall,/area/crew_quarters/bar)
+"bam" = (/obj/structure/sign/barsign{pixel_y = 0},/turf/simulated/wall,/area/hallway/primary/starboard)
```
Comment (Menshin) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: Is the area change intentional ?

**Comment by Niknakflak** [comment]
Been meaning to remake this PR. Thanks!
Means I don't have to struggle with git on this different computer.

**Comment by Niknakflak** [comment]
If the barsign is changed to the single object instead of the double sprite decal, then it will need to be changed on metastation and ministation too I think.  I did that in the old PR but stopped at ministation because I ran into issues and then gave up.

**Comment by Mandurrrh** [comment]
I changed them on meta and box! And Menshin Ill message you on irc!

**Comment by Mandurrrh** [comment]
Also niknak Im glad you are happy about it. I had asked a few times and you werent available. I figured I waited a decent amount of time before finishing it up :+1:

**Comment by Niknakflak** [comment]
Yea, it's great.  Thanks and good job.

**Comment by Niknakflak** [comment]
Griefing is spelled wrong and its kind of an OOC term and then everything else looks great.

**Comment by Mandurrrh** [comment]
Its only for a right click description not posted or anything

**Comment by Niknakflak** [comment]
People will see it in game when examining though right?

File: code/game/objects/structures/barsigns.dm [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+	icon = "thesingulo"
+	desc = "You feel yourself gravitating towards this bar"
```
Comment (Miauw62) [inline code/game/objects/structures/barsigns.dm]: "Where people go that'd rather not be called by their name"

**Comment by Dorsisdwarf** [comment]
Make the sign empable/emaggable. When it's emagged/emped it flashes stupid colors and scrolls OPERATIVE really fast across it.

More things in the game should do funny but pointless things when you mess around with them.

**Comment by dannno** [comment]
removing "grief" was pretty unnecessary when it's a word already referenced in other aspects of the game
niknak is just autism
like 90% of the examine messages for stuff is OOC referencing

**Comment by optimumtact** [comment]
are you rage quitting or unfucking your branch? I would like to see this through to completion ifyou're willing to share the code

**Comment by Mandurrrh** [comment]
unfucking branch :)

File: code/game/objects/structures/barsigns.dm (hunk 4 above)
Comment (GunHog) [inline code/game/objects/structures/barsigns.dm]: :+1: for that desc change

**Comment by Niknakflak** [comment]
Ok sorry, but it is your character examining them sometimes.  I thought most had IC descriptions.

**Comment by Mandurrrh** [comment]
I took it out Niknak

**Comment by Niknakflak** [comment]
was responding to danno, sorry if the previous comment sounded pushy ;_;
looks good to merge

**Comment by Chocobro** [comment]
barsignsemag
http://turd.church/ss13/shit-i-made/misc/barsignsemag.dmi

**Comment by GunHog** [comment]
That should honestly be the EMP response.

**Comment by xxalpha** [comment]
Yes that should be emp_act and emag_act can be done later if you can't get a proper sprite for it.

**Comment by WJohn** [comment]
:+1: These look pretty good.

**Comment by Niknakflak** [comment]
Those look great, I also didn't even realize "the greytide" sign was a different sprite from bays.  It looks fantastic.

**Comment by Mandurrrh** [comment]
I sent them to you originally and you said they weren't ss13 themed. Im glad you like them now!

**Comment by Steelpoint** [comment]
They look great :+1:

**Comment by Niknakflak** [comment]
The only one I ever saw before was a honked sign without a logo.

File: code/game/objects/structures/barsigns.dm (hunk 4 above)
Comment (Miauw62) [inline code/game/objects/structures/barsigns.dm]: You didn't change the desk! ;-;

**Pull request merged**